zip-0252.rst

```diff
@@ -190,9 +184,9 @@ pre-NU5 nodes will gradually be disconnected in the run up to activation.
 Support in Zebra
 ================
 
-Partial support for NU5 on Testnet will be implemented in Zebra version
-1.0.0-alpha.18, which will advertise protocol version 170015. This version
-will validate a strict subset of NU5 consensus rules on Testnet.
+Partial support for NU5 on Testnet will be implemented in Zebra version [TODO],
+which will advertise protocol version 170050. This version will validate a
+strict subset of NU5 consensus rules on Testnet.
```

Comment:
```suggestion
Several versions of Zebra have implemented versions of the NU5 consensus
rules on Testnet:

- `zebrad` v1.0.0-alpha.18 implemented partial support for NU5 on Testnet,
  validating a strict subset of the NU5 consensus rules. This version had an
  activation height of 1599200, as described in section _`NU5 deployment`_
  above.
- `zebrad` v1.0.0-beta.7 will fully validate the NU5 consensus rules. This is
  expected to be the final revision of the NU5 consensus rules. As part of these
  consensus rule changes, `zebrad` v1.0.0-beta.7 will automatically re-download
  the entire chain from genesis, then follow an alternate chain starting at height
  1599200. It will advertise protocol version 170050.

Support for NU5 on Mainnet will be implemented in `zebrad` version [TODO],
which will advertise protocol version 170100.
```